pytorch_lightning/accelerators/legacy/tpu_accelerator.py

```diff
@@ -307,29 +305,6 @@ def load_spawn_weights(self, original_model):
 
         return loaded_model
 
-    def transfer_distrib_spawn_state_on_fit_end(self, model, mp_queue, results):
```

Comment:
Is this removed for a reason? I thought we didn't want to change any legacy files and then remove them completely

Reply:
probably someone accidently working in wrong tpu accelerator file. Since this file won't be used anyway it should be okay, right?